_posts/2022-12-16-emacs-package.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: "Emacs Package"
+date: 2022-12-16 18:10:00 +0000
+---
+
+Boxes integration is now available as an Emacs package 😊<br/>
+[![MELPA](https://melpa.org/packages/boxes-badge.svg)]({{ site.baseurl }}/editors-emacs.html)
```

Comment:
I was first confused that the MELPA link didn't bring me to MELPA, so I would suggest that we have two links: one to MELPA and another one to our own page.
```suggestion
[Boxes integration for Emacs]({{ site.baseurl }}/editors-emacs.html) is now available as an Emacs package!<br/>
[![MELPA](https://melpa.org/packages/boxes-badge.svg)](https://melpa.org/#/boxes)
```

Reply:
Damn 100% failure rate 🤢

If it's confusing then of course I'll change it.

Thanks,
Mike

Reply:
> Damn 100% failure rate

What I see is that you have a 100% *success* rate!

Reply:
Looks good Thomas,

Thanks,
Mike